rwengine/src/audio/SoundManager.cpp

```diff
@@ -87,6 +91,18 @@ void SoundManager::initializeAVCodec() {
 #endif
 }
 
+bool SoundManager::initializeEFX() {
+    if (!alcIsExtensionPresent(alDevice, ALC_EXT_EFX_NAME)) {
+        RW_ERROR("EFX extention is not present");
+        return false;
+    }
+
+    initEfxFunctionPointers();
+
+    SoundEffect effect(0);
```

Comment:
Have no idea what zero mean... Maybe define some constants?

Reply:
It's just a test code, deleted.